suppressible-error-prone/src/test/java/com/palantir/suppressibleerrorprone/WhitespaceIndentBeforeTest.java

```diff
@@ -43,6 +43,28 @@ void something_else_is_before_on_the_same_line() {
         assertThat(whitespaceIndentBefore("class Foo {} |class Bar {}")).isEqualTo(" ");
     }
 
+    @Test
```

Comment:
I think it's nice to use junit5's `@Nested` here to group together tests for entirely different methods - also means you don't need to prefix the test names with "start position". [Example](https://github.com/palantir/gradle-guide/blob/a626b8e1cfeddcd1c34decde5712e4955adae2a0/gradle-guide-error-prone/src/test/java/com/palantir/gradle/guide/errorprone/ConfigurationAvoidanceRegistrationTest.java#L109)

Reply:
I didn't know about it - this is pretty nice 👍